src/components/App/App.tsx

```diff
@@ -55,7 +55,7 @@ function App() {
       apiCalls.fetchCountriesData()
       .then((data) => setCountries(data))
     }
-    
+   // try using an empty array on line 59 after the closing curly. This may let us get rid of the conditional! 
```

Comment:
this was one of my takeaways from the hooks lesson today as well! We should ask instructors if the conditional is the best way to use useEffect here to solve the infinite loop issue